### PR TITLE
Close bug #802 and maybe #1388

### PR DESCRIPTION
**Pull request by johnlepikhin**

### CODE REVIEW

**Comment by johnlepikhin** [comment]
> All checks have failed

There is some bug in myocamlbuid.ml generator; I manually removed Lazy.force from generated file and that was enough for successful compilation.

File: src/plugins/internal/InternalInstallPlugin.ml [hunk 1]
```diff
@@ -262,6 +262,19 @@ let install pkg argv =
             in
 
             let acc =
+              (* Add findlibextra files *)
+              let path =
+                OASISHostPath.of_unix bs.bs_path
+              in
```
Comment (gildor478) [inline src/plugins/internal/InternalInstallPlugin.ml]: Strange indentation, I think you use tabs here. Please remove.

**Comment by gildor478** [comment]
Is this possible to add a test ?

**Comment by gildor478** [comment]
https://github.com/ocaml/oasis/commit/f0b13cbfa7350f412b8c2ca3336dc0c9214db482

**Pull request closed without merging**